pkg/kube/inject/validate.go

```diff
@@ -163,7 +163,7 @@ func splitPorts(portsString string) []string {
 func parsePort(portStr string) (int, error) {
 	port, err := strconv.ParseUint(strings.TrimSpace(portStr), 10, 16)
 	if err != nil {
-		return 0, fmt.Errorf("failed parsing port '%d': %v", port, err)
+		return 0, fmt.Errorf("failed parsing port '%s': %v", portStr, err)
```

Comment:
What problem is this solving?

Reply:
> What problem is this solving?

if err is not nil, port will be always `0` , so we should return the origin `string` format, but not parsed `int` format